spec/abstractops.html

```diff
@@ -648,6 +638,22 @@ <h1>LargerOfTwoTemporalDurationUnits ( _u1_, _u2_ )</h1>
     </emu-alg>
   </emu-clause>
 
+  <emu-clause id="sec-temporal-mergelargestunitoption" aoid="MergeLargestUnitOption">
+    <h1>MergeLargestUnitOption ( _options_, _largestUnit_ )</h1>
+    <p>
+      The abstract operation MergeLargestUnitOption returns a new plain Object with the properties copied from the _options_ Object, and the `largestUnit` property set to _largestUnit_.
+    </p>
+    <emu-alg>
+      1. Let _merged_ be ! OrdinaryObjectCreate(%Object.prototype%).
+      1. Let _keys_ be ? EnumerableOwnPropertyNames(_options_, ~key~).
+      1. For each element _nextKey_ of _keys_, do
+        1. Let _propValue_ be ? Get(_options_, _nextKey_).
+        1. Perform ? Set(_merged_, _nextKey_, _propValue_, *true*).
+      1. Perform ? Set(_merged_, *"largestUnit"*, _largestUnit_, *true*).
```

Comment:
Why Set here? I'd expect to not invoke random Object.prototype setters that users have installed:
```suggestion
        1. Perform ! CreateDataPropertyOrThrow(_merged_, _nextKey_, _propValue_, *true*).
      1. Perform ! CreateDataPropertyOrThrow(_merged_, *"largestUnit"*, _largestUnit_, *true*).
```